src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String[] cookieValues;
+  public Cookie userCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "loginStatus";
+    cookieValues = {"out", "null"};
```

Comment:
For the value, if we're storing the userId in our cookie, then we don't need multiple values. It'll either be the actual userId, or some constant to specify that the user is not  logged in (eg. zero or -1 are common).  We maybe also don't need to default the value here.